src/git-code-review

```diff
@@ -135,9 +151,6 @@ function main() {
             shift 1
             clear_all "$@"
             ;;
-        "")
```

Comment:
Took this out since it wasn't working _at all_, and my initial caveman logic is always "run a command and see what happens" and it kept yelling at me with errors.  So having it skip this and go to usage makes it easier to remember just what to do.

Just like if you typed `git` with no args.